bin/integration_test.sh

```diff
@@ -69,8 +69,8 @@ docker-compose up -d antenna
 docker-compose run -u "${HOSTUSER}" test ./bin/aws_s3.sh rm "${DESTBUCKET}" --recursive
 
 # Get a crash id from the fakecrashdata directory
-CRASHID="11107bd0-2d1c-4865-af09-80bc00220228"
-CRASHKEY="v2/raw_crash/111/20220228/${CRASHID}"
+CRASHID="11107bd0-2d1c-4865-af09-80bc00220909"
+CRASHKEY="v1/raw_crash/20220909/${CRASHID}"
```

Comment:
I don't know why I decided it was a good idea to change the date, but I did. That made this PR a bit longer.